backend/src/tools/api.py

```
@@ -184,37 +184,30 @@ def make_api_call_func(
         data: Optional request body data for POST/PUT requests
         path_params: Optional dictionary of path parameters to format into the URL
     """
-    async def api_call(event: str, data: Dict[str, Any], config: RunnableConfig):
+    async def api_call(event: str, data: Dict[str, Any] = {}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)